src/lib/rest/rest.cpp

```diff
@@ -1298,7 +1298,7 @@ static int connectionTreat
     //
     // Transaction starts here
     //
-    lmTransactionStart("from", ip, port, url);  // Incoming REST request starts
+    lmTransactionStart("from", "", ip, port, url);  // Incoming REST request starts
```

Comment:
In the case of incoming connection is not so simple... Can we assume that we should always "https" when `-https` is being used and  "http" otherwise? Maybe the client tried to use "http" but ended using "https" after a  3xx redirect..

I'm not sure and, in case of doubt, I prefer to leave the schema open using "" here.